src/BaseFirestoreRepository.spec.ts

```diff
@@ -11,8 +11,7 @@ export class Band {
   formationYear: number;
   lastShow: Date;
   genres: Array<string>;
-  @SubCollection(Album)
-  albums?: ISubCollection<Album>;
+  @SubCollection(Album) albums?: ISubCollection<Album>;
```

Comment:
Can you return this new line? The Decorator looks weird if is in the same line

Reply:
@wovalle Done!